diff --git a/data/com.github.muriloventuroso.pdftricks.appdata.xml.in b/data/com.github.muriloventuroso.pdftricks.appdata.xml.in
index aa5836a..6a555a0 100644
--- a/data/com.github.muriloventuroso.pdftricks.appdata.xml.in
+++ b/data/com.github.muriloventuroso.pdftricks.appdata.xml.in
@@ -5,10 +5,9 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_group>MuriloVenturoso</project_group>
   <project_license>GPL-3.0+</project_license>
-  <value key="x-appcenter-suggested-price">1</value>
-  <value key="x-appcenter-stripe">pk_live_8nDudUPTmiwzRlzvKulsBlVm</value>
   <name>PDF Tricks</name>
   <summary>Tricks for PDF Files</summary>
+  <launchable type="desktop-id">com.github.muriloventuroso.pdftricks.desktop</launchable>
   <description>
     <p>A simple, efficient application for small manipulations in PDF files. </p>
     <ul>
@@ -172,6 +171,7 @@
   <custom>
     <value key="x-appcenter-color-primary">#F6546A</value>
     <value key="x-appcenter-color-primary-text">#fff</value>
+    <value key="x-appcenter-stripe">pk_live_8nDudUPTmiwzRlzvKulsBlVm</value>
     <value key="x-appcenter-suggested-price">2</value>
   </custom>
 </component>
